src/vs/workbench/browser/parts/editor/media/tabstitlecontrol.css

```diff
@@ -121,6 +123,16 @@
 	flex-shrink: 0;
 }
 
+.monaco-workbench .part.editor > .content .editor-group-container > .title .tabs-container > .tab.sizing-fixed {
+	min-width: var(--tab-sizing-current-width, 50px);
+	max-width: var(--tab-sizing-current-width, var(--tab-sizing-fixed-max-width, 160px));
+	flex: 1 0 0; /* all tabs are evenly sized and grow */
+}
+
+.monaco-workbench .part.editor > .content .editor-group-container > .title > .tabs-and-actions-container.wrapping .tabs-container > .tab.sizing-fixed.last-in-row:not(:last-child) {
+	flex-grow: 0; /* prevent last tab in a row from moving to next row when tab widths are fixed */
```

Comment:
Can you clarify why this is needed? I am not sure I see any impact in my testing.

Reply:
Thanks, that's been removed now, I thought it was helping but I no longer think so.